fractal/fractal.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+/*
+* Require the path module
+*/
+const path = require('path');
+
+/*
+ * Require the Fractal module
+ */
+const fractal = module.exports = require('@frctl/fractal').create();
+
+/*
+ * Give your project a title.
+ */
+fractal.set('project.title', 'Startup');
+
+/*
+ * Tell Fractal where to look for components.
+ */
+fractal.components.set('path', path.join(__dirname, '../docroot/themes/custom/startup_theme/assets/src/components'));
```

Comment:
Components need to live on the theme root, not in the `assets` directory.

Reply:
Paths in this file also need changing when the Fractal config & install moves to the theme directory.